pl_bolts/models/detection/faster_rcnn/backbones.py

```diff
@@ -0,0 +1,36 @@
+import torch.nn as nn
+
+from pl_bolts.models.detection.components import create_torchvision_backbone
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+try:
+    from torchvision.models.detection.backbone_utils import resnet_fpn_backbone
+except ModuleNotFoundError:
+    warn_missing_pkg('torchvision')  # pragma: no-cover
+
+
+def create_fasterrcnn_backbone(backbone: str, fpn: bool = True, pretrained: str = None,
+                               trainable_backbone_layers: int = 3, **kwargs) -> nn.Module:
```

Comment:
Can we have docstring for `trainable_backbone_layers` as well?

Reply:
We have :)